codespell_lib/data/dictionary.txt

```diff
@@ -20069,6 +20069,7 @@ prersistent->persistent
 presance->presence
 prescrition->prescription
 prescritions->prescriptions
+presearve->preserve
```

Comment:
So can we also have preserved, preservation, preserving and preserver please.

Reply:
Of course :) Thanks for these ideas